### ISSUE TITLE
Packages file for debian broken

### ISSUE THREAD

**Issue by skalderqm** [adoptium/adoptium-support#931]
### Please provide a brief summary of the bug

https://packages.adoptium.net/artifactory/deb/dists/bookworm/main/binary-amd64/Packages
https://packages.adoptium.net/artifactory/deb/dists/bullseye/main/binary-amd64/Packages

contain:

Filename: pool/main/t/temurin-${VERSION}/temurin-21-jdk_21.0.1.0.0+12_amd64.deb
Filename: pool/main/t/temurin-${VERSION}/temurin-17-jdk_17.0.9.0+9_amd64.deb
Filename: pool/main/t/temurin-${VERSION}/temurin-11-jdk_11.0.21.0.0+9_amd64.deb

that makes apt-get install fail:

# apt-get install temurin-17-jdk
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
The following packages will be upgraded:
  temurin-17-jdk
1 upgraded, 0 newly installed, 0 to remove and 2 not upgraded.
Need to get 166 MB of archives.
After this operation, 130 kB of additional disk space will be used.
Err:1 https://packages.adoptium.net/artifactory/deb bullseye/main amd64 temurin-17-jdk amd64 17.0.9.0+9
  400  Bad Request [IP: 151.101.67.42 443]
E: Failed to fetch https://packages.adoptium.net/artifactory/deb/pool/main/t/temurin-${VERSION}/temurin-17-jdk_17.0.9.0%2b9_amd64.deb  400  Bad Request [IP: 151.101.67.42 443]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?

### Expected Results

installation of packages should succeed

### Actual Results

installation of packages fails

### What Java Version are you using?

N/A

**Comment by ustndagsemih** [comment]
Hello,
We are also having the same issue.

**Comment by thecow275** [comment]
I get the same 400 error when trying to update my installed Temurin packages with apt on debian.  <html><br></br>Package download failed:<br/><br/>E: https://packages.adoptium.net/artifactory/deb bookworm/main amd64 temurin-11-jdk amd64 11.0.21.0.0+9 is not (yet) available (400  Bad Request [IP: 151.101.247.42 443])
E: https://packages.adoptium.net/artifactory/deb bookworm/main amd64 temurin-17-jdk amd64 17.0.9.0+9 is not (yet) available (400  Bad Request [IP: 151.101.247.42 443])
</html>

**Comment by raylandaalman** [comment]
I got the same error too.

Err:1 https://packages.adoptium.net/artifactory/deb jammy/main amd64 temurin-21-jdk amd64 21.0.1.0.0+12
  400  Bad Request [IP: 146.75.79.42 443]
E: Failed to fetch https://packages.adoptium.net/artifactory/deb/pool/main/t/temurin-${VERSION}/temurin-21-jdk_21.0.1.0.0%2b12_amd64.deb  400  Bad Request [IP: 146.75.79.42 443]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?

In my troubleshooting, I re-downloaded the Adoptium GPG key, then removed and re-added the Adoptium apt repository. Instructions I followed are at:
https://adoptium.net/installation/linux/

This is affecting my Ubuntu system running 22.04, Jammy Jellyfish.

I have a Fedora system, which had no issues with adding the Adoptium RPM repo, and installing.
